src/ontology/uberon-edit.obo

```diff
@@ -224223,6 +224223,14 @@ intersection_of: part_of UBERON:0007318 ! saphenous vein
 relationship: dc-contributor https://orcid.org/0000-0002-0819-0473 ! Paula Duek Roggli
 property_value: dcterms-date "2023-08-31T11:41:09Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480069
+name: temporofacial region
+def: "The region at each side of the face comprising the temple and the cheek." [https://orcid.org/0000-0002-0819-0473]
```

Comment:
Since there is not a dbxref and I could not find one myself after a cursory search, I am not sure if this term should comprise the cheek. Looking at this image, the cheek would be part of the zygomatic region. Is the zygomatic region part of the temporofacial region?
![image](https://github.com/obophenotype/uberon/assets/94959119/0720631b-3d2d-4ebe-a297-16a1ce04c626)

Reply:
@aleixpuigb Many thanks for reviewing the term. We need temporofacial region to to map the tissue that originated the cell line Muraoka (CVCL_E042) described in PMID:1357816. Agree that the term is not widely used. I found it also in PMID:28448262. There is no definition in these papers, but according to image in PMID:28448262 the zygomatic region is part of the temporofacial region.

Reply:
@aleixpuigb Do you think it may be useful to add "this term is used to map the tissue that originated the cell line Muraoka (CVCL_E042) described in PMID:1357816" somewhere? If yes, where? curator note?